jobApplicationTracker/Models/JobApplication.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace jobApplicationTrackerApi.Models
+{
+    public class JobApplication
+    {
+        [Key]
+        public int Id { get; set; }    //Could be [Guid] instead of [int]
+
+        [MaxLength(100, ErrorMessage = "Company name could not exceed 100 characters")]
+        public string Company { get; set; } = string.Empty;
+
+        [MaxLength(100, ErrorMessage = "Job Title could not exceed 100 characters")]
+        public string JobTitle { get; set; } = string.Empty;
+        [Required]
+        public int StatusId { get; set; }  //could be [Guid]
+        public DateTime? ApplicationDate { get; set; }   //class diagram want it to be not null, but if we are planning to create a canban board if it will be in a so called TODO table it should be null in my opinion. 
+        public DateTime? InterviewDate { get; set; }
+        public string? Notes { get; set; }
+        public decimal? Salary { get; set; }
```

Comment:
Do you need Currency? SalaryType  (hourly/monthly) etc?

Reply:
Great point, Larry! Adding **Currency** and **SalaryType** (e.g., _hourly_, _monthly_) could make the Salary field more descriptive and flexible. I'll update the model to include these fields so that salary data can be more precise. Thanks for the suggestion!